sdk/metricsadvisor/ai-metrics-advisor-rest/src/metricsAdvisorKeyCredentialPolicy.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest,
+} from "@azure/core-rest-pipeline";
+import { KeyCredential } from '@azure/core-auth';
+export const API_KEY_HEADER_NAME = "Ocp-Apim-Subscription-Key";
+export const X_API_KEY_HEADER_NAME = "x-api-key";
+
+/**
+ * Interface parameters for updateKey function
+ */
+export interface MetricsAdvisorKeys extends KeyCredential {
+  /** API key from the Metrics Advisor web portal */
+  // key?: string;
```

Comment:
@joheredi Do you think that we rename the property name from `key` to `apiKey` in `KeyCredential` would be more proper?